hawkular-metrics/hawkular-metrics-wrapper.sh

```diff
@@ -91,16 +91,46 @@ fi
 sed -i "s|<!-- ##ADDITIONAL LOGGERS## -->|$additional_loggers|g" ${JBOSS_HOME}/standalone/configuration/standalone.xml
 
 # Setup the truststore so that it will accept the OpenShift cert
+KEYTOOL_COMMAND=/usr/lib/jvm/java-1.8.0/jre/bin/keytool
 HAWKULAR_METRICS_AUTH_DIR=$HAWKULAR_METRICS_DIRECTORY/auth
 mkdir $HAWKULAR_METRICS_AUTH_DIR
 pushd $HAWKULAR_METRICS_AUTH_DIR
 
-cp $KEYSTORE hawkular-metrics.keystore
-cp $TRUSTSTORE hawkular-metrics.truststore
+if [ ! -n "$KEYSTORE" ]; then
```

Comment:
Is there any reason why we should be checking for this? I think it makes sense to just require that the certificate and keys are passed in, and don't give the option of passing in a keystore